src/platform_impl/linux/wayland/seat/pointer/handlers.rs

```diff
@@ -0,0 +1,333 @@
+//! Handlers for the pointers we're using.
+
+use std::cell::RefCell;
```

Comment:
This file doesn't exist in the tree, would suggest to simply remove your changes and add them again.

Reply:
Done, let me know if I messed up something.